DailyGoals/DailyGoals/Entities/TeamMember+Extensions.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Team.swift
+//  DailyGoals
+//
+//  Created by Alisa Bagrova on 12/12/2022.
+//
+
+import Foundation
+
+extension TeamMember {
+
+	static let all: [TeamMember] = [
```

Comment:
why are there so many spaces everywhere @kishnel

Reply:
Mmm if you’re referring to the indentation spaces, that could be because on my personal laptop I’m using tabs instead of spaces, while on our work laptops Xcode is configured to always use spaces